### DISCUSSION TITLE
How can I use multiline edit in ideavim?

### DISCUSSION THREAD

**Discussion by Kevin1118** [JetBrains/ideavim#383] [Q&A]
I use ideavim in MacOS.

I can hit `shift + v` to enter `visual-line` mode, but when I hit `shift + i` try to edit multiline, I failed.

I have checked my Goland Keymap, try to find shortcut `shift + i`, but nothing to show. This mean there is not keymap conflict.

So, how can I use multiline edit in ideavim?

**Comment by AlexPl292** [comment]
Hi, vim doesn't have multiline editing after shift-v. You can modify multiple lines after ctrl-v  and shift-i.
